airbyte-integrations/bases/base-java/src/test/java/io/airbyte/integrations/base/AirbyteTraceMessageUtilityTest.java

```diff
@@ -51,6 +55,16 @@ void testEmitErrorTrace() {
     assertJsonNodeIsTraceMessage(Jsons.deserialize(outContent.toString(StandardCharsets.UTF_8)));
   }
 
+  @Test
+  void testCorrectStacktraceFormat() {
+    try {
+      int x = 1 / 0;
+    } catch (Exception e) {
+      AirbyteTraceMessageUtility.emitSystemErrorTrace(e, "you exploded the universe");
+    }
+    Assertions.assertTrue(outContent.toString(StandardCharsets.UTF_8).contains("\\n\\tat"));
```

Comment:
This is a really basic check for the format, ideally we align this with the regex we build in the parser @pedroslopez but since this is coming first I'm just checking for the new line, tab, _at_